app/scripts/oshinko.js

```diff
@@ -1,11 +1,121 @@
 'use strict';
 
 angular.module('openshiftConsole')
+  .factory('ClusterActions', function ($uibModal) {
+    function deleteCluster(clusterName) {
+      var modalInstance = $uibModal.open({
+        animation: true,
+        controller: 'ClusterDeleteCtrl',
+        templateUrl: '/views/' + 'delete-cluster.html',
+        resolve: {
+          dialogData: function () {
+            return {clusterName: clusterName};
+          }
+        }
+      });
+
+      modalInstance.result.then(function () {
+        console.info("Modal completed");
+      }, function () {
+        console.info('Modal dismissed at: ' + new Date());
+      });
+    }
+
+    function newCluster() {
+      var modalInstance = $uibModal.open({
+        animation: true,
+        controller: 'ClusterNewCtrl',
+        templateUrl: '/views/' + 'new-cluster.html',
+        resolve: {
+          dialogData: function () {
+            return {};
+          }
+        }
+      });
+
+      modalInstance.result.then(function () {
+        console.info("Modal completed");
+      }, function () {
+        console.info('Modal dismissed at: ' + new Date());
+      });
+    }
+
+    function scaleCluster(cluster, workerCount) {
+      var modalInstance = $uibModal.open({
+        animation: true,
+        controller: 'ClusterDeleteCtrl',
+        templateUrl: '/views/' + 'scale-cluster.html',
+        resolve: {
+          dialogData: function () {
+            return {
+              clusterName: cluster,
+              workerCount: workerCount
+            };
+          }
+        }
+      });
+
+      modalInstance.result.then(function () {
+        console.info("Modal completed");
+      }, function () {
+        console.info('Modal dismissed at: ' + new Date());
+      });
+    }
+
+    return {
+      deleteCluster: deleteCluster,
+      newCluster: newCluster,
+      scaleCluster: scaleCluster,
+    };
+  })
+  .factory('clusterData', [
+    '$http',
+    '$q',
+    "OSHINKO_CFG",
+    function ($http, $q, OSHINKO_CFG) {
+      console.log(OSHINKO_CFG.restPort);
```

Comment:
how to get rest url value in console ?
the config.json is only for dev setup, doesnt exists in console.